egs2/ksponspeech/asr1/conf/decode_asr.yaml

```diff
@@ -0,0 +1,3 @@
+lm_weight: 0.6
+ctc_weight: 0.4
+beam_size: 60
```

Comment:
The beam size is too large, and it would take a too long time for inference.
Please test it with a smaller beam size (10 or 20).
If the performance degradation is not significant, we can use this smaller beam size config.

Reply:
Equivalent performance was obtained by beam size of 10, so fixed accordingly.